ibtk/src/math/PETScVecUtilities.cpp

```diff
@@ -623,10 +623,9 @@ PETScVecUtilities::constructPatchLevelAO_cell(AO& ao,
                                               Pointer<PatchLevel<NDIM> > patch_level,
                                               const int ao_offset)
 {
-    int ierr;
     if (ao)
     {
-        ierr = AODestroy(&ao);
+        const int ierr = AODestroy(&ao);
```

Comment:
Not a big deal, but I don't like to make these `const`, because you then have to go back and remove the qualifier if you add in more PETSc calls.

Reply:
(BTW, this is the same reason that `ierr` is scoped the way that it is.)